notifier.cpp

```diff
@@ -100,28 +123,30 @@ int main(int argc, char** argv) {
     natsOptions_Destroy(opts);
     stanConnOptions_Destroy(connOpts);
 
-    bool is_warn = false;
-    while (s == NATS_OK) {
-        auto msg = std::make_unique<message>();
-        std::getline(std::cin, msg->data);
+    fill_msgs_queue();
```

Comment:
On start, there will be buffered records.
Why there is no attempt to send thems?